extensions/amp-fit-text/1.0/component.js

```diff
@@ -70,20 +70,21 @@ export function FitText({
   }, [resize]);
 
   return (
-    <ContainWrapper
-      size={true}
-      layout={true}
-      paint={true}
-      ref={containerRef}
-      wrapperClassName={classes.fitTextContentWrapper}
-      contentRef={measurerRef}
-      contentClassName={classes.fitTextContent}
-      {...rest}
-    >
-      <div ref={heightRef} className={classes.minContentHeight}>
-        {children}
-      </div>
-    </ContainWrapper>
+    <Wrapper {...rest}>
```

Comment:
Any way to reverse these wrappers? It'd be very good to have `contain: size layout paint` on the top wrapper.

Reply:
Done